README.md

```diff
@@ -71,6 +71,25 @@ conda install qtpy
 ```
 
 
+### Mypy
+
+A CLI is offered to help with usage of QtPy.
+Presently, the only feature is to generate command line arguments for Mypy
+that will enable it to process the QtPy source files with the same API
+as QtPy itself would have selected.
```

Comment:
Maybe add here that the example result below is the one you get when running QtPy with PySide2 as the available/selected binding.

Reply:
Also, checking this inside an env without bindings installed probably we should also add somewhere that the default output for this command is `--always-true=PYQT5 --always-false=PYQT6 --always-false=PYSIDE2 --always-false=PYSIDE6` ?

Reply:
Good point, the existing description here was not very good, and rather incomplete and potentially confusing. I expanded it to explicitly mention both those things as well as the actual example CLI invocation and other clarifications.